eng/scripts/Create-APIView.ps1

```diff
@@ -185,18 +213,18 @@ function New-SwaggerAPIViewTokens {
   }
   LogGroupEnd
 
-  # Get Related Swagger ReadMe Files
-  $swaggerReadMeFiles = [System.Collections.Generic.HashSet[string]]::new()
+  # Get Related AutoRest Configuration Information
+  $autoRestConfigInfo = [System.Collections.Generic.Dictionary[string, object]]::new()
   $changedSwaggerFiles | ForEach-Object {
-    $readmeFile = Get-SwaggerReadMeFile -swaggerFile $_
-    if ($readmeFile) {
-      $swaggerReadMeFiles.Add($readmeFile) | Out-Null
+    $configInfo = Get-AutoRestConfigInfo -swaggerFile $_
+    if ($null -ne $configInfo -and -not $autoRestConfigInfo.ContainsKey($configInfo.ConfigPath)) {
+      $autoRestConfigInfo[$configInfo.ConfigPath] = $configInfo
     }
   }
 
   LogGroupStart " Swagger APIView Tokens will be generated for the following configuration files..."
-  $swaggerReadMeFiles | ForEach-Object {
-    LogInfo " - $_"
+  $autoRestConfigInfo.GetEnumerator() | ForEach-Object {
```

Comment:
If this was just a list you could just show the .ConfigPath property. That would make it a little easier to understand what this Key represents.